temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -229,3 +230,187 @@ message GetRegionResponse {
     // The temporal cloud region.
     temporal.api.cloud.region.v1.Region region = 1;
 }
+
+message GetIncomingServicesRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100. 
+    int32 page_size = 1;
+
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
+
+    // Filter incoming services by the namespace they route to - optional.
+    string namespace = 3;
+
+    // Filter incoming services by the task queue they route to - optional.
+    string task_queue = 4;
+
+    // Filter incoming services by their name - optional. Specifying this will result in zero or one results.
+    string name = 5;
+}
+
+message GetIncomingServicesResponse {
+    // The list of incoming services in ascending ids order
```

Comment:
Looks like you stopped using periods for sentences from here on down.
I'm very much in favor of always using periods in comments.

Reply:
just an accident